ando/tools/generator/AnDOGenerator.py

```diff
@@ -264,9 +296,12 @@ def generate_Struct(csv_file, pathToDir):
     Essential information of the following attributes needs to be present
     {ESSENTIAL_CSV_COLUMNS}
 
-    Args:
-        csv_file ([csv file ]): [Csv file that contains a list of directories to create]
-        pathToDir ([Path to directory]): [Path to directory where the directories will be created]
+    Parameters
+    ----------
+        csv_file : csv file
+            Csv file that contains a list of directories to create
+        pathToDir : Path to directory
+            Path to directory where the directories will be created
```

Comment:
We should also refactor `pathToDir` as it is not following the PEP8 standard (no CamelCase for variables, but underscores)
```suggestion
    Parameters
    ----------
    csv_file: str
        Csv file that contains a list of directories to create.
    pathToDir: str
        Path to directory where the directories will be created.
```